model/user-agent/registry.yaml

```
@@ -35,3 +35,21 @@ groups:
           using a user-agent for non-browser products, such as microservices with multiple names/versions inside the
           `user_agent.original`, the most significant version SHOULD be selected. In such a scenario it should align
           with `user_agent.name`
+      - id: user_agent.synthetic.type
+        stability: experimental
+        brief: >
+         Specifies the category of synthetic traffic, such as monitoring, crawler, bot, or another automation.
```

when reading this, I was expecting the enum to have values for `monitoring`, `crawler`, and maybe `other`, maybe could keep it simple
```suggestion
         Specifies the category of synthetic traffic, such as tests or bots.
```